networking/manager.go

```diff
@@ -2145,8 +2145,12 @@ func InitNetworkManager(networkconfig *maestroSpecs.NetworkConfigPayload, ddbcon
 
 	//Setup the config with the given network config
 	if inst.networkConfig != nil {
-		log.MaestroInfof("NetworkManager: Submit config read from config file\n")
-		inst.submitConfig(inst.networkConfig)
+		if inst.networkConfig.Disable == false {
```

Comment:
What happens if the `disable` flag doesn't exist in the config?

We need to guarantee backwards compatibility with all of the old configs that are out in the field

Reply:
Yes, I took into consideration backward compatibility and that's why added a disable flag instead of enable flag. If the disable flag does not exist, it is considered to be false by default. 
The maestro networking tests running right now also do not have the disable flag in their respective config options and are still passing. So I think we should be fine.